css/ding_library_opening_hours.css

```diff
@@ -43,11 +43,11 @@
  */
 @font-face {
```

Comment:
Why do we need to define the font-face here?

Reply:
Because this module requires Fontawesome fonts which are present in module's root folder.

Reply:
But aren't these fonts are also placed and defined in `themes/ddbasic`? I don't think they should be defined again.

Reply:
No. In themes/ddbasic are used Icomoon and package use here doesn't contain all icons which are used in project.

Reply:
Then I think we should adjust to Icomoon. Which icons are used in this module, and which do you think is missing?

Reply:
Problem is that the "core" Icomoon package contains very limited amount of icons. We can remove Fontawesome fonts from this module but we anyway will have to place here "font" folder with Icomoon fonts generated specially for this module.

Reply:
I'm not sure I understand this. Icomoon is a webfont like Fontawesome. When the font is loaded you can call any of the 490 icons with `content:`.

Reply:
Nope. Not in this case. Icomoon in current realization is used like a customized package and contains ~30 icons. So each time we want to add new icon in "core" forn package we have to replace files in "fonts/" folder of theme and also change @font-face declaration in scss file, because for each regeneration of font files also are changing hashes of attached files (https://github.com/easySuite/ding2/blob/development-4.x/themes/ddbasic/sass/base/icomoon.scss#L8 - **see "?pehhuu"** part)

Reply:
Wow, thats insane. Thank you for clarifying.